### PR TITLE
MM-41835 Add docs for EnableClientPerformanceDebugging setting

### PR DESCRIPTION
**Pull request by hmhealey**
Hopefully these changes are self-explanatory

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-41835

#### Related Pull Requests
https://github.com/mattermost/mattermost-webapp/pull/9875

### CODE REVIEW

File: source/configure/configuration-settings.rst [hunk 1]
```diff
@@ -1928,6 +1928,19 @@ Enable Developer Mode
 | This feature's ``config.json`` setting is ``"EnableDeveloper": false`` with options ``true`` and ``false``. |
 +-------------------------------------------------------------------------------------------------------------+
 
+Enable Client Performance Debugging
```
Comment (cwarnermm) [inline source/configure/configuration-settings.rst:1931]: Is a Telemetry page update needed as well for this new config setting?

**Review by cwarnermm — COMMENTED** [review]
Thanks for this great docs update! I've left a question about telemetry updates. Also -- is this config setting being released as part of the v6.5 release? Please specify a milestone as part of this PR so that it's clear when to merge it into the codebase. Thanks!

File: source/configure/configuration-settings.rst (hunk 1 above)
Comment (hmhealey) [reply source/configure/configuration-settings.rst:1931]: Added. I also fixed a random x in those docs that seemed like a mistake.

**Review by cwarnermm — APPROVED** [review]
Thank you so much!

**Comment by cwarnermm** [comment]
Given that there's also a new setting for end users to enable/disable, are you open to documenting that update as well, @hmhealey?

**Comment by hmhealey** [comment]
Yeah, I could add docs for that. Where would we put docs for that sort of setting? I'm not aware of any docs for the Account Settings stuff

**Comment by cwarnermm** [comment]
That would be fantastic, @hmhealey!!  The end-user configuration details would go on [this](https://docs.mattermost.com/messaging/manage-channels-settings.html#advanced) page, immediately following [this](https://docs.mattermost.com/messaging/manage-channels-settings.html#preview-pre-release-features) subsection, since the page is organized to match the UI.

**Comment by hmhealey** [comment]
Re-requesting review since I added the new section as part of this PR. I also bumped the milestone since it's not making it in for 6.5 at this point

**Review by cwarnermm — APPROVED** [review]
Thank you so much for developing this product documentation, @hmhealey!

**Pull request merged**